src/ImageTransformations.jl

```diff
@@ -93,4 +93,3 @@ center{T,N}(img::AbstractArray{T,N}) = SVector{N}(map(_center, indices(img)))
 _center(ind::AbstractUnitRange) = (first(ind)+last(ind))/2
 
 end # module
-
```

Comment:
No worries, I'm *really* unconcerned about things like whitespace :smile:. It's only the fact that I have a setting in emacs to automatically get rid of it that this got caught. You're helping a lot!